blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -3,19 +3,23 @@ import Immutable from 'immutable';
 since we would return a function, */
 /* eslint new-cap: [0] */
 const { Record } = Immutable;
-export const PipelineRecord = Record({
+export class PipelineRecord extends Record({
```

Comment:
For reference, it is, see the end of these docs: https://facebook.github.io/immutable-js/docs/#/Record